tools/devctr/pyproject.toml

```diff
@@ -7,11 +7,13 @@ authors = ["Your Name <you@example.com>"]
 [tool.poetry.dependencies]
 aws-embedded-metrics = "^3.1.0"
 black = "^23.3.0"
-boto3 = "^1.26.140"
-botocore = "^1.29.140"
 gitlint = "^0.19.1"
+ipython = "^8.15.0"
```

Comment:
> We pin numpy to version 1.24.2, as versions newer than this cause pytho
nto crash with SIGILL on AL2 4.14 c7g.metal instances. I expect this
will be patched in the upcoming 1.25.3 release.

The commit message has a typo `nto`